vrs/vrs_annotation_batch.py

```diff
@@ -246,15 +250,15 @@ def main(args):
         # Import all annotated shards
         ht_annotated = hl.import_vcf(
             annotated_file_list,
-            reference_genome="GRCh38",
+            reference_genome=assembly,
         ).make_table()
         logger.info("Annotated table constructed")
 
         vrs_struct = hl.struct(
-            VRS_Allele=ht_annotated.info.VRS_Allele,
-            VRS_Start=ht_annotated.info.VRS_Start,
-            VRS_End=ht_annotated.info.VRS_End,
-            VRS_Alt=ht_annotated.info.VRS_Alt,
+            VRS_Allele_IDs=ht_annotated.info.VRS_Allele.split(","),
+            VRS_Starts=ht_annotated.info.VRS_Start.split(",").map(lambda x: hl.int(x)),
+            VRS_Ends=ht_annotated.info.VRS_End.split(",").map(lambda x: hl.int(x)),
+            VRS_States=ht_annotated.info.VRS_Alt.split(","),
```

Comment:
does their updated code already output these with the correct names now, such as "VRS_Ends" instead of "VRS_End"?

Reply:
ype, the code in the most recent version does use 'VRS_Starts' and 'VRS_Ends', I'll be sure to include it in the next round of edits